### PR TITLE
Loosen the requirements

### PR DESCRIPTION
**Pull request by araffin**
Otherwise pip will complain if you have newer version.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 from setuptools import setup
 
 install_requires = [
-    "huggingface_hub==0.8.1",
-    "pyyaml==6.0",
+    "huggingface_hub~=0.8.1",
```
Comment (simoninithomas) [inline setup.py:4]: For huggingface_hub I'm not sure the tilde is useful since there is no version 0.8 above 0.8.1

**Review by simoninithomas — CHANGES_REQUESTED** [review]
Hey there 👋 thanks for the PR 🤗 , just for Hugging Face Hub there is no need of tilde since 0.8.1 is the highest 0.8 version

File: setup.py (hunk 1 above)
Comment (araffin) [reply setup.py:4]: oh, I meant `~=0.8` (if hf follows semantic versioning)

Comment (osanseviero) [reply setup.py:4]: 0.8.0 was yanked as there was a backwards compatible issue so it's not installable. 0.8.1 is the only version in the 0.8 range, so adding this change would not contribute here

Comment (araffin) [reply setup.py:4]: I meant `~=0.8` to allow `0.9`, `0.x.x`

Comment (osanseviero) [reply setup.py:4]: Ah I see! Could you modify that in the PR please? :pray:

**Review by simoninithomas — APPROVED** [review]
LGTM

**Comment by simoninithomas** [comment]
Awesome thanks, I'm merging it 🥳

**Pull request merged**